test/controllers/demo_mang_controller_test.rb

```diff
@@ -48,11 +48,11 @@ class DemoMangControllerTest < ActionDispatch::IntegrationTest
           end
 
           it 'should define render_authenticate_user_error' do
-            assert @controller.methods.include?(:render_authenticate_user_error)
+            assert @controller.methods.include?(:render_authenticate_error)
           end
 
           it 'should define render_authenticate_mang_error' do
-            assert @controller.methods.include?(:render_authenticate_mang_error)
+            assert @controller.methods.include?(:render_authenticate_error)
```

Comment:
Also here: The same note about "tests' contents do not match their descriptions".